### PR TITLE
Remove event aggregator

### PR DESCRIPTION
**Pull request by siwane**
This PR aims to delete eventAggregator.

After investigation, this service aggregate events and iterate on it to emit them. It also defer emission that is a breaking change, especially for core_kernel_classes_Resource

### CODE REVIEW

File: views/package-lock.json [hunk 1]
```diff
@@ -39,9 +39,9 @@
             "integrity": "sha512-S6/2ol3NpT99su9ZlZgoVvZHKW+yRVro562Ab1pizyCwv2YfO1r2FbpEDTS/cgfqI2JvgMtuP1V1SjR3T8Avrg=="
         },
         "@oat-sa/tao-core-ui": {
-            "version": "1.10.1",
-            "resolved": "https://registry.npmjs.org/@oat-sa/tao-core-ui/-/tao-core-ui-1.10.1.tgz",
-            "integrity": "sha512-tj5pUc99ETtnilNgcd2uDnDvNDe0PEupc6y62X168QcwM2wO+h2T+UarxTuT6l0G2DFdlrNCsz4A4O7Db2PHAA=="
+            "version": "1.10.5",
```
Comment (bartlomiejmarszal) [inline views/package-lock.json:42]: What is this change for?

Comment (siwane) [reply views/package-lock.json]: fixed

**Review by bartlomiejmarszal — APPROVED** [review]
LGTM

**Review by gabrielfs7 — APPROVED** [review]
LGTM

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request closed without merging**